src/electron.ts

```diff
@@ -116,9 +115,11 @@ app.on('activate', () => {
 
 function registerShortcuts(win: Electron.BrowserWindow): void {
```

Comment:
Yeah, I like that idea. It'll make much easier to modify shortcuts, I'll do that tomorrow...